private-templates-service/client/src/ErrorMessage.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Alert } from "reactstrap";
+
+interface Props {
```

Comment:
Might want to think about what naming conventions you are going to start using.  Maybe 'ErrorMessageProps', since this will be able to be accessed outside this file